app/views/orders/_order_line_item.erb

```diff
@@ -0,0 +1,15 @@
+<tr>
+  <td>
+    <div class="d-flex mb-2">
+      <div class="flex-shrink-0">
+        <%= image_tag order_line_item.meal.image, width:'35', class:'img-fluid' if order_line_item.meal.image.attached? %>
+      </div>
+      <div class="flex-lg-grow-1 ms-3">
+        <h6 class="small mb-0"><a href="#" class="text-reset"><%= order_line_item.meal.name %></a></h6>
+        <span class="small"><%= order_line_item.meal.non_veg? ? 'Non-Veg' : 'Veg' %></span>
```

Comment:
this logic can be required at multiple places. Move this inside meal.

Reply:
Done